src/Core/Framework/Resources/config/packages/dev/shopware.yaml

```diff
@@ -12,5 +15,6 @@ shopware:
 
     usage_data:
         gateway:
-            dispatch_enabled: false
-            base_uri: 'http://localhost:8888'
+            dispatch_enabled: true
+            entity_dispatch_enabled: false
+            consent_dispatch_enabled: true
```

Comment:
Question: Should we also make these configurable via env variables? This would enable us to skip that in the acceptance tests :smile:

Reply:
I would prefer to have a yaml file like this to disable it. I think their APP_ENV is e2e, so create an e2e folder and disable it there

Reply:
I double checked this and we are using `APP_ENV=prod` and therefore it won't be possible to create a new config for that specific env. Do you have another idea because I would also not like to make these configs configurable via env :see_no_evil:

Reply:
We changed it to make it configurable via env variables as it is not possible to create a new config for a different env as we are still using `APP_ENV=prod`